### PR TITLE
BUG: cdf load

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses changes in pysat.

If a file can't be loaded properly, the cdf load routine may sometimes fail. This guarantees that if a load fails, an empty dataset will be returned.

Note that this does not fix the root issue in #211, but will at least let the code proceed as intended

# Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

pytest with pysat develop

**Test Configuration**:
* Operating system: Monterrey
* Version number: Python 3.10.9
* pysat develop

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by jklenzing** [comment]
pinging @JonathonMSmith for review

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   * Fix general clean routine to skip transformation matrices
   * New window needs to be integer for calculate_imf_steadiness
   * Fixed version import
+  * Fixed a bug when data fails to load for cdf pandas objects
```
Comment (aburrell) [inline CHANGELOG.md:15]:
```suggestion
  * Fixed a bug when data fails to load for CDF pandas objects
```

File: pysatNASA/tests/test_methods_cdaweb.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def test_load_with_empty_file_list(self):
```
Comment (aburrell) [inline pysatNASA/tests/test_methods_cdaweb.py:49]: Recommend adding a `@pytest.mark.parameterize(pandas_format, [True, False])` to this test.  I also think you need to add a `pysat.Meta()` to the `cdw.load_xarray` function return on line 369.

Also recommend adding a test for `cdw.load_pandas` with a bad filename that will trigger the 'unable to load' logger warning and cover line 274 (return empty data and meta).

**Review by aburrell — CHANGES_REQUESTED** [review]

File: pysatNASA/tests/test_methods_cdaweb.py (hunk 2 above)
Comment (jklenzing) [reply pysatNASA/tests/test_methods_cdaweb.py:51]: Updated. Note that the file needs to exist for this warning to trigger. Added an empty file to load.

Just this once, i threw in an extra unit test for another uncovered warning.

**Review by aburrell — APPROVED** [review]
The cdaweb testing is almost into the green now!

**Pull request merged**